src/lib/components/button/QBtn.svelte

```diff
@@ -149,8 +150,8 @@
   <span class="q-btn__label">
     {#if label}
       {label}
-    {:else}
-      {@render children?.()}
     {/if}
+
```

Comment:
This change will probably mean we can have both a `label` and contents set, which might not be desirable.

Reply:
Yes but it's up to the user to not break things? Or maybe we can make `children` overwrite `label`? Might work too but is pretty bad DX when using tooltips

Reply:
The safer solution would be better here imo, as it's not super clear that you are not supposed to use both the slot and the `label` attribute.